types/workspace.ts

```diff
@@ -1,5 +1,6 @@
 import { URI, TextDocument } from './lsp'
 
+/** Server to client. Used to ask the user to select a folder, if the workspace folder is too big */
```

Comment:
"Server to client" is specified in README - I would prefer to keep it there, specifying the direction of request in one place only looks inconsistent to me.
"if the workspace folder is too big" - this is just one of examples when it will be used, I do not think we need it in comment like this.
"select a folder" -> "select files(s) or folder(s)"